apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -9,8 +9,18 @@
             "type"  : "array",
             "items" : {
                 "type"    : "string",
-                "pattern": "^http://[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*:[0-9]+/$", 
-                "example" : "http://a.b.c:123/"
+                "pattern": "^http://[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*:[0-9]+/$",
+                "example" : "http://somehost.somedomain:123/"
+            }
+        },
+        "blocked_peers" : {
+            "description" :
+            "Pre-configured addresses of epoch nodes NOT to contact",
+            "type"  : "array",
+            "items" : {
+                "type"    : "string",
+                "pattern": "^http://[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*:[0-9]+/$",
```

Comment:
This type seems meant to be the same as peers key above, and complex enough to warrant a type. Though this is only refactoring so not needed for this PR.